.gitignore

```diff
@@ -11,3 +11,5 @@ build
 coverage
 node_modules
 test/benchmarks/.tmp
+coverage.lcov
+test-suite-output.tap
```

Comment:
I can't see where in the new tests this file is generated. Is it just a leftover from a refactor or is it a Jenkins file that is being generated locally in some scenarios or something like that?

Reply:
That's generated when running the `.ci/scripts/test.sh` script locally:

```bash
$ .ci/scripts/test.sh 12
...
node_tests_1     | ---------------------------------|---------|----------|---------|---------|-------------------------
node_tests_1     | + nyc report --reporter=lcov
docker_node_tests_1 exited with code 0
Aborting on container exit...
+ NODE_VERSION=12
+ docker-compose --no-ansi --log-level ERROR -f .ci/docker/docker-compose-all.yml down -v
...

$ ls -l test-suite-output.tap coverage.lcov
-rw-r--r--  1 vmartinez  staff       0  3 Mar 14:21 coverage.lcov
-rw-r--r--  1 vmartinez  staff  303892  3 Mar 14:21 test-suite-output.tap
```